students.sql

```diff
@@ -6,6 +6,7 @@ SELECT
 FROM custom.vw_enrollment_course_section_ps cs
 LEFT JOIN custom.lkSchools lks
     ON cs.schoolid_record = lks.SystemSchoolID
+    AND cs.CurrGradeLevel BETWEEN lks.LowGrade AND lks.HighGrade
```

Comment:
Might be useful to have a version of the following in the join to support on the K-8 schools, depending on how they need things printed moving forward  

AND lks.id NOT IN ('8', '9', '21', '22', '20') -- Excludes KECP and Valiant Lower, Upper and Bridge K-8

Reply:
Good call out. We're ok here without it because there is a WHERE filter on SchoolName_Tableau, and that column is unique across all the dupe school records in lkschools. (eg. "Bridge K-8" vs "Bridge Rising" vs "Bridge Upper").